config.docker-template.php

```diff
@@ -97,4 +97,13 @@
     define('TEST_ENROL_LDAP_DOMAIN', 'ou=Users,dc=openstack,dc=org');
 }
 
+if (getenv('BBBMOCK')) {
```

Comment:
```suggestion
if (getenv('MOODLE_DOCKER_BBB_MOCK')) {
```

I'd suggest to use the very same name and then add it to the `bbb-mock.yml`file. See above.

Reply:
Agree - thanks.